core/src/main/java/org/openstack4j/openstack/storage/block/internal/BlockVolumeServiceImpl.java

```diff
@@ -159,6 +160,27 @@ public VolumeType createVolumeType(VolumeType volumeType) {
         return post(CinderVolumeType.class, uri("/types")).entity(volumeType).execute();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public EncryptionType createEncryptionType(EncryptionType encryptionType, String volumeTypeId) {
+        checkNotNull(encryptionType);
+        checkNotNull(volumeTypeId);
+        return post(CinderEncryptionType.class, uri("/types/%s/encryption", volumeTypeId))
+                .entity(encryptionType).execute();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void deleteEncryptionType(String encryptionId, String volumeTypeId) {
+        checkNotNull(encryptionId);
+        checkNotNull(volumeTypeId);
+        delete(Void.class, uri("/types/%s/encryption/%s", volumeTypeId, encryptionId)).execute();
+    }
+
```

Comment:
Can you please also implement `getEncryptionType`? This was it can only be manipulated, but not queried.

Reply:
Done, I have implemented getEncryption and committed it.

Reply:
@olivergondza Kindly review the latest commit made with modifications.